examples/orthographic.js

```diff
@@ -0,0 +1,92 @@
+/* global itowns, document, renderer */
```

Comment:
If you do `let renderer` here, you don't need it in global. Also I'm suprised to see `document` here. eslint knows about `document` being global :-)